numpy/lib/_arraysetops_impl.py

```diff
@@ -414,12 +414,15 @@ def unique_all(x):
     """
     Find the unique elements of an array, and counts, inverse and indices.
 
-    This function is an Array API compatible alternative to:
+    This function is an Array API compatible alternative to `np.unique`, but
+    returns a namedtuple for easier access to each output.
 
+    Example
```

Comment:
The naming (especially) and the ordering of the docstring sections is important and is specified by the [numpydoc style guide](https://numpydoc.readthedocs.io/en/latest/format.html).

The sections must be named `Examples` and preferably would be moved back to the original location in the docstring.

Reply:
Hey @rossbar , I've made the requested changes to follow the NumPy docstring style guide, renaming 'Example' to 'Examples' and restoring the original order of sections. Please review the updated PR. Thanks!